Framework/DataHandling/inc/MantidDataHandling/LoadILLDiffraction.h

```diff
@@ -64,7 +64,7 @@ class MANTID_DATAHANDLING_DLL LoadILLDiffraction
   void fillDataScanMetaData(const NeXus::NXDouble &);
   void fillMovingInstrumentScan(const NeXus::NXUInt &, const NeXus::NXDouble &);
   void fillStaticInstrumentScan(const NeXus::NXUInt &, const NeXus::NXDouble &,
-                                const NeXus::NXFloat &);
+                                const double &);
```

Comment:
It does not make much sense to pass primitive scalar by reference, can you pass by value instead?